client/src/app/videos/+video-watch/comment/video-comment-add.component.html

```diff
@@ -20,3 +22,24 @@
     </button>
   </div>
 </form>
+
+<ng-template #visitorModal let-modal>
+  <div class="modal-header">
+    <h4 class="modal-title" id="modal-basic-title" i18n>You are one step away from commenting</h4>
+    <button type="button" class="close" aria-label="Close" (click)="hideVisitorModal()"></button>
+  </div>
+  <div class="modal-body" i18n>
+    <span i18n>
+      If you have an account on this instance, you can <a (click)="gotoLogin()" i18n>login to comment</a>.
+    </span>
+    <span i18n>
+      Otherwise you can comment using an account on an ActivityPub-compatible instance:
+    </span>
+    <my-remote-subscribe interact="true" account="{{ uri }}"></my-remote-subscribe>
```

Comment:
If you want to represent a boolean, it should be `[interact]="true"` (in your current code, it will be the string "true")